### PR TITLE
API Camera Feed Implementation

### PR DESCRIPTION
**Pull request by ivanvuong**
-- Implemented Camera Feed to the Screen Two Console 
-- Created helpers that call HoloLens API for the video and returned styled video in the component 

For now, undefined is returned, which is why there is a blank screen. Commented out original camera feed function in 2 different places, **exporting in @camera_feed.txt, and **import&calling in @/screen_two/page.tsx, instead implementing youtube video rendering 

-- Made a basic "TempYoutubeVideo" function that renders a youtube video. 

-- Commented out original code in page.tsx regarding Telemetry Page 

UNSURE of general name of function in page.tsx (remains "TelemetryPage()")
UNSURE of formatting of the video: size, etc

Current implementation of Screen Two (Youtube Video)
Screenshot 2024-02-09 224620

Commented out implementation of the API Camera Feed 
Screenshot 2024-02-09 224656

### CODE REVIEW

File: client/components/hmd_link/camera_feed.tsx
Comment (abhi-arya1) [inline client/components/hmd_link/camera_feed.tsx:65]: use TailwindCSS styling for this (look @ map component to see how that works) since that is our project convention

File: client/components/hmd_link/camera_feed.tsx
Comment (abhi-arya1) [inline client/components/hmd_link/camera_feed.tsx:14]: leaving this comment here so you can read it--don't worry ab it tho

File: client/components/hmd_link/camera_feed.tsx
Comment (abhi-arya1) [inline client/components/hmd_link/camera_feed.tsx:74]: make sure to not have `export` and `export default` in the same file. I've fixed this this time around, but letting you know for nxt time

File: client/app/screen_two/page.tsx
Comment (abhi-arya1) [inline client/app/screen_two/page.tsx:16]: there's an emergent issue with the map (yet again). i'll be opening an issue. We'll merge this PR with that `<Map />` line commented out as such: `{ /* <Map /> */ }`.

**Review by abhi-arya1 — CHANGES_REQUESTED** [review]
take a look at comments--we'll go over final stuff in the code review, but looks good and works rn 😄

File: client/components/ui/map.tsx
Comment (abhi-arya1) [inline client/components/ui/map.tsx]: you ran into a merge conflict here--again, i've fixed it for now, but make sure to `git merge main` *before* your PR next time

**Review by abhi-arya1 — COMMENTED** [review]
everything looks good and works well, but there's an issue with the map flexbox (that's not on your end but on the integration of all the components--once that's fixed then we can merge

**Review by abhi-arya1 — APPROVED** [review]
map fixed, LGTM (do not merge, a co-lead will do this)

**Review by Turfader — APPROVED** [review]
I trust that this doesn't break anything. Good luck

**Pull request merged**
